cadl-tests/generate.sh

```diff
@@ -15,19 +15,29 @@ function generate {
     rm -rf cadl-output
 }
 
+mkdir existingcode
+cp -rf ./src existingcode
```

Comment:
just copy src/main/java/com/cadl/partialupdate?

Reply:
I found it's easier to copy all the code to maintain the `src/main/java/` folder structure, but can `mkdir` the folders and then copy as well.

Reply:
Please focus on the source file that is required in your test case, and not involve others that not related in your test case.